### ISSUE TITLE
Grease pencil line art modifier Edge Types panel - update tooltips for greyed out props in

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#3196]
... and give a hint why they are greyed out and what to do to make them active. We have quite a few of them. No idea how to activate them. There is also the Allow Overlapping Type prop that is activated, but deactivates under some circumstances that i have yet to find out. The code is an if ...

![howotoactivate](https://user-images.githubusercontent.com/13919702/184527407-59489f3f-fc8e-4bc6-a0b5-68981a9ba1c9.jpg)

Related: Left align grease pencil lineart modifier UI after merge #3113

**Comment by ReinerBforartists** [comment]
Blender Manual is not updated to this changes yet. Postponing.

https://docs.blender.org/manual/en/dev/grease_pencil/modifiers/generate/line_art.html

**Comment by ReinerBforartists** [comment]
As an info: they turn active when you have a light source defined in the light reference panel below.

**Comment by Draise14** [comment]
Postponed for 3.6.0

**Comment by Draise14** [comment]
![image](https://user-images.githubusercontent.com/35357409/231616319-9ebcf226-1265-47db-8332-fdef4956ee0b.png)

For clarity sake, shouldn't these be inside the light reference sub-panel? Since.. it's dependant on that?

**Comment by ReinerBforartists** [comment]
Yeah, but it belongs to the edges types. We could hide it instead of greying out. But this could also lead to confusion.

**Comment by Draise14** [comment]
Been thinking about this one more, maybe a sub-label for these two and the tooltip update but keeping them in position might help here.

**Comment by ReinerBforartists** [comment]
A label shounds like no bad idea

**Comment by ReinerBforartists** [comment]
C code :'3

**Comment by ReinerBforartists** [comment]
Part 1
![step 1](https://github.com/Bforartists/Bforartists/assets/13919702/223fa69a-1dac-4f6c-a264-98c5e6ab605f)

**Comment by ReinerBforartists** [comment]
Let's have a look if we can put the illumination filtering also under the Light Object label

**Comment by ReinerBforartists** [comment]
Part 2

Create prop needs intendation. And a decorator off to have a gap at the end.

![part2](https://github.com/Bforartists/Bforartists/assets/13919702/2d376651-3ea4-4601-a55d-f87e5b4ef7e3)

**Comment by ReinerBforartists** [comment]
And i think we better put cast shadow below Light Contour. It is not dependant of Light Contour ...

**Comment by ReinerBforartists** [comment]
As close as i can get. Documenting.

![closeenough](https://github.com/Bforartists/Bforartists/assets/13919702/b5bf1c64-9684-4881-8a84-6d96828ac004)

**Comment by ReinerBforartists** [comment]
Done

And finally this dinosaur is resolved.
